platform/security/servlet/security-servlet-logout/src/main/java/org/codice/ddf/security/servlet/logout/LocalLogoutServlet.java

```diff
@@ -57,6 +64,20 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
                 SecurityTokenHolder savedToken = (SecurityTokenHolder) session.getAttribute(
                         SecurityConstants.SAML_ASSERTION);
                 if (savedToken != null) {
+
+                    for (SecurityToken token : savedToken.getRealmTokenMap()
+                            .values()) {
+                        try {
+                            Subject subject = securityManager.getSubject(token);
```

Comment:
Same comment here, use SecurityAssertionImpl, it will be faster and also avoid a service reference.